understanding/conformance.html

```diff
@@ -546,56 +545,56 @@ <h3>Advisory Techniques for Conformance Claims</h3>
             In other words, the access issue must cause a proportionately greater problem for
             people with disabilities than it causes people without disabilities in order to be
             considered an accessibility issue (and covered under these accessibility guidelines).
          </li>
          
-         <li>All Success Criteria must also be testable. This is important since otherwise it would
-            not be possible to determine whether a page met or failed to meet the Success Criteria.
-            The Success Criteria can be tested by a combination of machine and human evaluation
-            as long as it is possible to determine whether a Success Criterion has been satisfied
+         <li>All success criteria must also be testable. This is important since otherwise it would
+            not be possible to determine whether a page met or failed to meet the success criteria.
+            The success criteria can be tested by a combination of machine and human evaluation
+            as long as it is possible to determine whether a success criterion has been satisfied
             with a high level of confidence.
          </li>
          
       </ol>
       
-      <p>The Success Criteria were assigned to one of the three levels of conformance by the
+      <p>The success criteria were assigned to one of the three levels of conformance by the
          working group after taking into consideration a wide range of interacting issues.
          Some of the common factors evaluated when setting the level included:
       </p>
       
       <ul>
          
-         <li>whether the Success Criterion is 
-            <strong>essential</strong> (in other words, if the Success Criterion isn't met, then even assistive technology
+         <li>whether the success criterion is 
+            <strong>essential</strong> (in other words, if the success criterion isn't met, then even assistive technology
             can't make content accessible)
          </li>
          
-         <li>whether it is possible to satisfy the Success Criterion for 
-            <strong>all Web sites and types of content</strong> that the Success Criteria would apply to (e.g., different topics, types of content,
+         <li>whether it is possible to satisfy the success criterion for 
+            <strong>all Web sites and types of content</strong> that the success criteria would apply to (e.g., different topics, types of content,
             types of Web technology)
          </li>
          
-         <li>whether the Success Criterion requires skills that could 
-            <strong>reasonably be achieved by the content creators</strong> (that is, the knowledge and skill to meet the Success Criteria could be acquired
+         <li>whether the success criterion requires skills that could 
+            <strong>reasonably be achieved by the content creators</strong> (that is, the knowledge and skill to meet the success criteria could be acquired
```

Comment:
Change to 'success criteron' to match the previous singular form earlier in the sentence.